examples/notebooks/submodel_cracking_DFN_or_SPM.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
I think this whole notebook can be updated to 
1. remove mention of `build=False` since you just need to select options
2. use the Ai2020 parameter set 
3. removed unused variables in the cell where the plot is made

Reply:
I was trying to show how to use the crack model with existing parameter sets. Do you think it is worthy of keeping it this way?

Reply:
I guess this comes back to the point of wanting to provide complete parameter sets for a cell, rather than mixing and matching. i.e. does it make sense to add these mechanical parameters to a different set?

Reply:
Okay, I will change the parameter set and remove the file in `pybamm/input/parameters/lithium-ion/mechanicals`